pyscf/scf/hf.py

```diff
@@ -225,10 +227,11 @@ def kernel(mf, conv_tol=1e-10, conv_tol_grad=None,
         norm_ddm = numpy.linalg.norm(dm-dm_last)
 
         conv_tol = conv_tol * 10
+        ddm_tol = ddm_tol * 10
         conv_tol_grad = conv_tol_grad * 3
         if callable(mf.check_convergence):
             scf_conv = mf.check_convergence(locals())
-        elif abs(e_tot-last_hf_e) < conv_tol or norm_gorb < conv_tol_grad:
+        elif abs(e_tot-last_hf_e) < conv_tol or norm_gorb < conv_tol_grad or norm_ddm < ddm_tol:
```

Comment:
I see that we are using norm_ddm which is numpy.linalg.norm(ddm - ddm_last), which is the Frobenius norm of "ddm - ddm_last". This is different from CFOUR which prints the largest density difference.